google/auth/credentials.py

```diff
@@ -26,6 +28,8 @@
 from google.auth._refresh_worker import RefreshThreadManager
 
 DEFAULT_UNIVERSE_DOMAIN = "googleapis.com"
+NO_OP_TRUST_BOUNDARY_LOCATIONS: "typing.Tuple[str]" = ()
```

Comment:
The lookup endpoint returns `"encodedLocations": "0x0"` if an account is not allow-listed, or trust boundary has not been setup. There is no `locations` field in the response. I got this response from the endpoint when testing, and confirmed it with Annan.

```
{
  "encodedLocations": "0x0"
}
```